### PR TITLE
docs: update with "Specifying Parser Options"

### PR DESCRIPTION
**Pull request by Ryan-Dia**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)
I added Supplementary explanations to the docs 

#### Is there anything you'd like reviewers to focus on?

Motivation

After reading the explanation, I thought that having a summary at the end would be helpful for better comprehension. Therefore, I added a summarizing section at the end of the text. This may assist readers who might have had difficulty understanding the earlier parts or those who need a more organized overview. Additionally, it will make the content easier to read and more reader-friendly.

I'm always grateful

### CODE REVIEW

**Review by nzakas — CHANGES_REQUESTED** [review]
Thanks. The additional sentence looks good, but you've also submitted a bunch of formatting changes. Can you please revert those?

File: docs/src/use/configure/language-options.md [hunk 1]
```diff
@@ -191,17 +190,17 @@ ESLint allows you to specify the JavaScript language options you want to support
...
 
-By the same token, supporting ES6 syntax is not the same as supporting new ES6 globals (e.g., new types such as `Set`). For ES6 syntax, use `{ "parserOptions": { "ecmaVersion": 6 } }`; for new ES6 global variables, use `{ "env": { "es6": true } }`. Setting `{ "env": { "es6": true } }` enables ES6 syntax automatically, but `{ "parserOptions": { "ecmaVersion": 6 } }` does not enable ES6 globals automatically.
+By the same token, supporting ES6 syntax is not the same as supporting new ES6 globals (e.g., new types such as `Set`). For ES6 syntax, use `{ "parserOptions": { "ecmaVersion": 6 } }`; for new ES6 global variables, use `{ "env": { "es6": true } }`. Setting `{ "env": { "es6": true } }` enables ES6 syntax automatically, but `{ "parserOptions": { "ecmaVersion": 6 } }` does not enable ES6 globals automatically. In summary, to support only ES6 syntax, use `{ "parserOptions": { "ecmaVersion": 6 } }`, and to support both ES6 syntax and new ES6 global variables, such as Set and others, use `{ "env": { "es6": true } }`.
```
Comment (fasttime) [inline docs/src/use/configure/language-options.md:193]:
```suggestion
By the same token, supporting ES6 syntax is not the same as supporting new ES6 globals (e.g., new types such as `Set`). For ES6 syntax, use `{ "parserOptions": { "ecmaVersion": 6 } }`; for new ES6 global variables, use `{ "env": { "es6": true } }`. Setting `{ "env": { "es6": true } }` enables ES6 syntax automatically, but `{ "parserOptions": { "ecmaVersion": 6 } }` does not enable ES6 globals automatically. In summary, to support only ES6 syntax, use `{ "parserOptions": { "ecmaVersion": 6 } }`, and to support both ES6 syntax and new ES6 global variables, such as `Set` and others, use `{ "env": { "es6": true } }`.
```

Just a suggestion to improve formatting.

**Comment by Ryan-Dia** [comment]
> Thanks. The additional sentence looks good, but you've also submitted a bunch of formatting changes. Can you please revert those?

I apologize. I have reverted the formatting changes to their original state.

**Review by nzakas — APPROVED** [review]
LGTM, thanks. Would like @fasttime to verify his comments were addressed before merging.

**Review by mdjermanovic — APPROVED** [review]

**Review by fasttime — APPROVED** [review]
LGTM, thanks!

**Pull request merged**